VirtualEconomyFramework/VENFTApp-Blazor/Models/TxDetails.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using VEDriversLite.NeblioAPI;
+
+public enum TxWay
+{
+    In,
+    Out,
+    Sub,
+}
+
+public class TxDetails
+{
+    public GetTransactionInfoResponse Info { get; set; } = new GetTransactionInfoResponse();
+    public string Sender { get; set; } = string.Empty;
+    public string Receiver { get; set; } = string.Empty;
+    public bool FromAnotherAccount { get; set; } = false;
+    public bool FromSubAccount { get; set; } = false;
+    public DateTime Time { get; set; } = DateTime.MinValue;
+    public TxWay Way
+    {
+        get
+        {
+            if (FromAnotherAccount)
+                return TxWay.In;
+            else if (FromAnotherAccount)
+                return TxWay.Sub;
+            return TxWay.Out;
+        }
+    }
+}
```

Comment:
I moved this into separated file + added `TxWay` enum

Reply:
Great. I will add this Dto later to the VEDriversLite. I am doing the refactoring of main logic in #33 so I will use the same structure to keep it compatible.